docroot/themes/custom/twentyseventeen/sass/components/footer/_footer.scss

```diff
@@ -0,0 +1,72 @@
+// Footer
+//
+// Secondary links, disclaimers and social links
+//
+// Markup: footer.twig
+//
+// Style guide: components.footer
+
+.footer {
+  padding: 2rem;
+  text-align: center;
+  background: $blu;
+  color: $lt-blu;
+
+  a {
+    color: $lt-blu;
+    display: inline-block;
+    transition: color 0.5s ease;
+
+    &:hover,
+    &:focus {
+      color: $wht;
+    }
+  }
+}
+
+.footer__links {
```

Comment:
so this is a minor point but you can leverage sass here a little instead of having to do .footer__links 

you could instead do
````
.footer {
   &__links {
   }
}
````

This will get compiled into .footer__links